dex/testing/eth/harness.sh

```diff
@@ -44,8 +44,6 @@ DELTA_NODE_KEY="725394672587b34bbf15580c59e5199c75c2c7e998ba8df3cb38cc4347d46e2b
 DELTA_ENODE="ca414c361d1a38716170923e4900d9dc9203dbaf8fdcaee73e1f861df9fdf20a1453b76fd218c18bc6f3c7e13cbca0b3416af02a53b8e31188faa45aab398d1c"
 DELTA_NODE_PORT="30307"
 DELTA_AUTHRPC_PORT="8555"
-export DELTA_HTTP_PORT="38556"
-export DELTA_WS_PORT="38557"
```

Comment:
Delta is a light wallet, so I think we should not use them for testing at the moment. This was probably done on Delta because the wallet will allow the open http/ws ports because an account is not unlocked. The full nodes are unlocked for mining meaning they panic if you try to set these ports in the config. However, we can use the authed port even if the geth node has an unlocked account.